docs/design/2021-09-22-data-consistency.md

```diff
@@ -0,0 +1,121 @@
+# Proposal: Reduce Data inconsistencies
+
+- Author(s): [Li Su](http://github.com/lysu), [Ziqian Qin](http://github.com/ekexium)
+- Tracking Issue: https://github.com/pingcap/tidb/issues/26833
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+
+- [Introduction](#introduction)
+- [Motivation or Background](#motivation-or-background)
+- [Detailed Design](#detailed-design)
+
+- [Test Design](#test-design)
+
+  - [Regression Tests](#regression-tests)
+  - [System Tests](#system-tests)
+  - [Performance Tests](#performance-tests)
+
+- [Impacts & Risks](#impacts--risks)
+
+- [Investigation & Alternatives](#investigation--alternatives)
+
+- [Unresolved Questions](#unresolved-questions)
+
+## Introduction
+
+The document proposes an enhancement that reduces data inconsistency issues in TiDB.
```

Comment:
How about also mentioning about inconsistent write operation detection(?), and richer diagnose information in case inconsistency is found?

Reply:
IMO the "inconsistent write operation detection" is already included in the description. 

I added a note about the enhancement in diagnosis info, but didn't go into details. Do you think it necessary?

Reply:
I think it's good enough now.